src/selectors.css

```diff
@@ -285,3 +286,13 @@
 @custom-selector :--volumeMounts [data-itemprop~='volumeMounts'];
 @custom-selector :--volumeMount [data-itemprop~='volumeMount'];
 @custom-selector :--volumeNumber [itemprop~='volumeNumber'];
+
+
+/**
+ * Group element selectors
+ */
+@custom-selector :--ListTypes :--Article:--root > :--affiliations,:--Collection,:--List,:--references > ol;
```

Comment:
Do you have a view on whether a it should be possible / allowable for a particular theme to define its own custom selectors, or whether you see this as a core thema fuction and so should only happen here?

Reply:
I think theme authors should have freedom to define similar selectors on a per theme basis if needed, given that they do so in a way that stays within the boundaries of the project goals (being a good CSS citizen when used in third-party contexts, and avoiding coupling of the selector targets to specific website HTML structure so as to keep the themes useful for multiple authors).

These selectors were the ones that I felt I was reaching for multiple times, and thought they would be useful across themes.

Reply:
+1 to "theme authors should have freedom to define similar selectors on a per theme basis if needed"